test/src/main/java/org/apache/accumulo/test/util/SerializationUtil.java

```diff
@@ -28,7 +28,7 @@
 import java.io.Serializable;
 import java.util.Objects;
 
-import org.apache.commons.codec.binary.Base64;
+import org.apache.accumulo.core.util.Base64;
```

Comment:
I had thought these both (commons and our built-in utility for base64) might have been replaced in 2.0 with the built-in Base64 decoder, but I can't recall.

Reply:
Yes, accumulo.core.util.Base64 has been dropped since 2.0.  Do you think it is worth it to replace all uses in this version?  Perhaps maybe a follow on task?

Reply:
To answer my own question.. it wasn't too bad.  I dropped the Accumulo Bas64 in
bcc8379